pom.xml

```diff
@@ -48,6 +48,24 @@
             <version>1.18.12</version>
             <scope>provided</scope>
         </dependency>
+           <dependency>
+                <groupId>org.openmrs.module</groupId>
+                <artifactId>metadatadeploy-api</artifactId>
+                <version>${metadatadeployVersion}</version>
+                <scope>provided</scope>
+            </dependency>
+             <dependency>
+            	<groupId>org.openmrs.module</groupId>
+            	<artifactId>dataexchange-api</artifactId>
+            	<version>${dataexchangeVersion}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.openmrs.module</groupId>
+                <artifactId>reporting-api</artifactId>
+                <version>${reportingVersion}</version>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
Do we actually need `reporting-api` as a dependency?

Reply:
Resolved thnks